srcpkgs/x86info/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'x86info'
+pkgname=x86info
+version=1.30
+revision=1
+only_for_archs="x86_64 x86_64-musl i686 i686-musl"
```

Comment:
Wouldn't `nocross` have the same affect as his?

Reply:
Nope. nocross would still allow to build it natively on e.g. arm*. This package should be only ever build able for a x86 target arch. But arm to x86 cross compilation theoretically could be allowed.

Reply:
Ah, right, only thought about how the builders build it here, thanks for correcting me here :)